src/eos-updater-poll-common.c

```diff
@@ -771,10 +786,11 @@ eos_update_info_to_string (EosUpdateInfo *update)
 
   results = g_string_free (g_steal_pointer (&results_string), FALSE);
 
-  return g_strdup_printf ("%s, %s, %s, %s\n   %s%s",
+  return g_strdup_printf ("%s, %s, %s, %s, %s\n   %s%s",
                           update->checksum,
                           update->new_refspec,
                           update->old_refspec,
+                          update->version,
```

Comment:
What if this is `NULL`?

Reply:
Good point. I can change it to `(NULL)`...